interfaces/cython/cantera/units/solution.py.in

```diff
@@ -0,0 +1,70 @@
+from .. import Solution as _Solution, PureFluid as _PureFluid, Heptane as _Heptane, CarbonDioxide as _CarbonDioxide, Hfc134a as _Hfc134a, Hydrogen as _Hydrogen, Methane as _Methane, Nitrogen as _Nitrogen, Oxygen as _Oxygen, Water as _Water
```

Comment:
You don't need to import `Heptane`, `CarbonDioxide`, etc. here, since you don't use them.